elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -740,56 +740,62 @@ elytron.token-realm.oauth2-introspection.client-ssl-context=The SSL context to b
-elytron.modifiable-security-realm.identity.clear.name=The credential name.
-elytron.modifiable-security-realm.identity.clear.password=The actual password to set.
-elytron.modifiable-security-realm.identity.set-password.simple-digest=A simple digest password.
-elytron.modifiable-security-realm.identity.set-password.simple-digest.algorithm=The algorithm used to encrypt the password.
-elytron.modifiable-security-realm.identity.set-password.simple-digest.password=The actual password to set.
-elytron.modifiable-security-realm.identity.simple-digest=A password in clear text.
-elytron.modifiable-security-realm.identity.simple-digest.name=The credential name.
-elytron.modifiable-security-realm.identity.simple-digest.algorithm=The algorithm used to encrypt the password.
-elytron.modifiable-security-realm.identity.simple-digest.password=The actual password to set.
-elytron.modifiable-security-realm.identity.set-password.salted-simple-digest=A password using a salted digest.
-elytron.modifiable-security-realm.identity.salted-simple-digest.name=The credential name.
-elytron.modifiable-security-realm.identity.set-password.salted-simple-digest.algorithm=The algorithm used to encrypt the password.
-elytron.modifiable-security-realm.identity.set-password.salted-simple-digest.salt=The salt to apply to the password.
-elytron.modifiable-security-realm.identity.set-password.salted-simple-digest.password=The actual password to set.
-elytron.modifiable-security-realm.identity.salted-simple-digest=A password using a salted digest.
-elytron.modifiable-security-realm.identity.salted-simple-digest.algorithm=The algorithm used to encrypt the password.
-elytron.modifiable-security-realm.identity.salted-simple-digest.salt=The salt to apply to the password.
-elytron.modifiable-security-realm.identity.salted-simple-digest.password=The actual password to set.
-elytron.modifiable-security-realm.identity.set-password.digest=A password using a salted digest.
-elytron.modifiable-security-realm.identity.set-password.digest.algorithm=The algorithm used to encrypt the password.
-elytron.modifiable-security-realm.identity.set-password.digest.realm=The realm.
-elytron.modifiable-security-realm.identity.set-password.digest.password=The actual password to set.
-elytron.modifiable-security-realm.identity.digest=A password using a salted digest.
-elytron.modifiable-security-realm.identity.digest.name=The credential name.
-elytron.modifiable-security-realm.identity.digest.algorithm=The algorithm used to encrypt the password.
-elytron.modifiable-security-realm.identity.digest.realm=The realm.
-elytron.modifiable-security-realm.identity.digest.password=The actual password to set.
+elytron.modifiable-security-realm.add-identity=Add an identity from a security realm.
+elytron.modifiable-security-realm.add-identity.identity=The name of the identity.
+elytron.modifiable-security-realm.remove-identity=Removes an identity from a security realm.
```

Comment:
Oh, I am sorry I have not expressed myself enough. The inconsistency is among conjugation of verbs in descriptions, e.g. "Removes an identity ..." vs "Remove an attribute" etc.

Just a cosmetic detail.